### PR TITLE
Migrate to typescript

### PR DESCRIPTION
**Pull request by thabarbados**
migrate to typescript
migrate tests to typescript
migrate stories to typescript
add tests for types

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -12,13 +12,5 @@
...
-  "plugins": [
-    [
-      "module:@import-meta-env/babel",
-      {
-        "example": "import.meta.env.VERSION"
-      }
-    ]
-  ],
   "comments": false
 }
```
Comment (dangreen) [inline .babelrc:16]: newline

Could you please create tasks for "codestyle" ? Need to create editorconfig and maybe renew linter/formatter.

File: rollup.config.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const plugins = (targets, vueOptions = {}) => [
   replace({
     preventAssignment: true,
     values: {
-      'import.meta.env.VERSION': JSON.stringify(pkg.version),
+      'process.env.version': JSON.stringify(pkg.version),
```
Comment (dangreen) [inline rollup.config.js:17]: please use CAPS

File: stories/AreaChart.stories.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { GChart } from 'vue-google-charts';
+import { GChart } from '../src/index';
```
Comment (dangreen) [inline stories/AreaChart.stories.ts:1]:
```suggestion
import { GChart } from '../src';
```

File: tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    /* Completeness */
+    "skipLibCheck": true
+  },
+  "include": [
+    "src"
+  ],
+  "exclude": [
+    "dist"
+  ]
+}
```
Comment (dangreen) [inline tsconfig.json:39]: ...

**Review by dangreen — CHANGES_REQUESTED** [review]

File: .babelrc (hunk 1 above)
Comment (thabarbados) [reply .babelrc]: Fixed. task created

File: rollup.config.js (hunk 2 above)
Comment (thabarbados) [reply rollup.config.js]: fixed

File: stories/AreaChart.stories.ts (hunk 3 above)
Comment (thabarbados) [reply stories/AreaChart.stories.ts]: fixed

File: tsconfig.json (hunk 4 above)
Comment (thabarbados) [reply tsconfig.json]: fixed

**Review by dangreen — APPROVED** [review]

**Pull request merged**